src/simpleValue.ts

```diff
@@ -0,0 +1,16 @@
+import DataLayerValue from './value';
```

Comment:
This represents just a simple value that doesn't change, which is what our domSource will be